Makefile

```diff
@@ -4,10 +4,17 @@ CLOUD9_DIR:=/var/lib/cloud9
 JEKYLL:=$(shell which jekyll)
 HTMLPROOFER:=$(shell which htmlproofer)
 NPM:=$(shell which npm)
+MRAA:=$(shell which mraa-gpio)
+EXDIR:= ./examples/extras/C-modulesFromJS/MRAA_Examples/
```

Comment:
If this needs to be built for packaging, it should be done in another Makefile, perhaps called by this one.

Reply:
I'm sorry, i am a little bit confused regarding your comments on the Makefile, presently i have added a Makefile in the examples directory which handles the compilation and that Makefile is called by this main /Makefile , for now i will revert the changes made to the main makefile and keep the Makefile in the examples directory as is.

Reply:
That's fine. The top-level Makefile isn't for inclusion on the target, it is for building the package that gets put on the target.